releasenotes/notes/fix-prometheus-scrape-types-508b16bcc24e3dc6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Allow prometheus scrape tls_verify to be set to false and 
```

Comment:
> tls_verify to be set to false
Just curious, why it was not possible to set it to `false` before?

Reply:
I believe it's because `false` is the zero value for `bool` in golang, so the field was omitted by the json marshaller if it was set to `false` (because of the `omitempty`). With `*bool` the zero value is now `nil`, so setting the field to `false` will be make the marshaller keep the value and only omit it if it's `nil`.